cmd/minikube/cmd/start.go

```diff
@@ -1779,3 +1781,61 @@ func exitGuestProvision(err error) {
 	}
 	exit.Error(reason.GuestProvision, "error provisioning guest", err)
 }
+
+func startNerdctld() {
+	// create a temporary directory
+	tmpDirName, err := os.MkdirTemp("/tmp", "minikube-nerdctld")
+	if err != nil && !os.IsExist(err) {
+		exit.Error(reason.IfSSHClient, "Error with creating "+tmpDirName, err)
+	}
+
+	// for containerd runtime using ssh, we need to install nerdctld and nerdctl within os
```

Comment:
add comment here that says will include in the ISO/Base image in the future versions